### ISSUE TITLE
txpool_content documentation seems wrong

### ISSUE THREAD

**Issue by tzapu** [ethereum/go-ethereum#16144]
Hello,

I think the documentation on https://github.com/ethereum/go-ethereum/wiki/Management-APIs#txpool_content
might be wrong, in my tests at transaction data level, parent nonce, the child is not an array but actual transaction

Sample return
```
{"jsonrpc":"2.0","id":1,"result":{"pending":{"0x001c7FAdfa3eA8252ecBF02F8B0Dd3013D80921b":{"380":{"blockHash":"0x0000000000000000000000000000000000000000000000000000000000000000","blockNumber":null,"from":"0x001c7fadfa3ea8252ecbf02f8b0dd3013d80921b","gas":"0x25e4c","gasPrice":"0x165a0bc00","hash":"0x346b5c38d8d9eb64647716488ff17d0e90e4438567c81b6434b8b57d0d1269bc","input":"0x","nonce":"0x17c","to":"0x45555629aabfea138ead1c1e5f2ac3cce2add830","transactionIndex":"0x0","value":"0x0","v":"0x26","r":"0xcd1b763a8a3bb844b56bc0b3db5729c1c77e7d07fbc4f0a6720dcc41685a3213","s":"0x678699ddeb4f998bfa321b38f8be7131a4786559f73459d9ca208cc3636e0f87"}},"0xe1105A19ea6d94f7b4572d1a2B9aF05C7bFA06E7":{"64":{"blockHash":"0x0000000000000000000000000000000000000000000000000000000000000000","blockNumber":null,"from":"0xe1105a19ea6d94f7b4572d1a2b9af05c7bfa06e7","gas":"0x20251","gasPrice":"0x4190ab00","hash":"0x24d92795e14e80e82ac1b911f4006aaa50d8f2bd205da741c77c752b5faaba06","input":"0x96b5a755000000000000000000000000000000000000000000000000000000000003b842","nonce":"0x40","to":"0xb1690c08e213a35ed9bab7b318de14420fb57d8c","transactionIndex":"0x0","value":"0x0","v":"0x26","r":"0xb96566f3753eef6ab0333b56210b30b4bdb6d7a33f46bf7c4eef89bea9bc072b","s":"0x7c42f87d436ab2d4da7cab75b83018be8346737dbdc31b2e44113d6ad1bbb8fe"}}}}}
```

#### System information

Geth version: `geth version` 1.8.1
OS & Version: Windows/Linux/OSX docker

#### Expected behaviour
return is mentioned to be the json representation of `map[string]map[string]map[string][]*RPCTransaction`
basically 
#### Actual behaviour
return is actually the json representation of `map[string]map[string]map[string]*RPCTransaction` 

#### Steps to reproduce the behaviour
make a rpc request
```
curl -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"txpool_content","params":[],"id":1}' http://localhost:8545
```

Cheers
